invenio_communities/views/decorators.py

```diff
@@ -11,18 +11,33 @@
 from functools import wraps
 
 from flask import g
+from flask_resources import CustomJSONSerializer, JSONSerializer, UIListSchema
 
+from invenio_communities.communities.resources.UISchema import \
+    UICommunitySchema
 from invenio_communities.proxies import current_communities
 
 
-def pass_community(f):
+def pass_community(serialize=False):
```

Comment:
I would avoid the default value `False`: this is because when using the decorator, if you don't pass the param (because your want it `False`), Flask will not be happy and will throw an error. It took me quite some time last time to debug it.
TL;DR: when a decorator as params, it must be always be called with the params.

Reply:
Ok, thanks a lot for sharing!